Update crud-how-to.md (#224)
Laravel-Backpack/docs: 4.1/crud-how-to.md

    public function index(Request $request)
    {
        $search_term = $request->input('q');
        

>         // NOTE: this is a Backpack helper that parses your form input into an usable array. 

Reviewer: ```suggestion
        // NOTE: You can use this Backpack helper to parse the form input into a usable array. 
```

>         // you still have the original request as `request('form')`

Reviewer: ```suggestion
        // but you still have the original request as `request('form')`
```

>         // NOTE: this is a Backpack helper that parses your form input into an usable array. 
>                  you still have the original request as `request('form')`

Reviewer 2: ```suggestion
        // NOTE: this is a Backpack helper that parses your form input into an usable array. 
        // you can always access the original request with `request('form')`
```
